source/sync/configure/sync-settings.txt

```diff
@@ -61,15 +61,63 @@ update the backend schema by changing the client model.
 To learn more about how the Realm Object Schemas map to the App Services Schemas
 when using Development Mode, refer to :ref:`Data Model Mapping <sync-data-model-mapping>`.
 
-For more information about modifying synced object schemas, including how to
-handle breaking changes, see: :ref:`<synced-schema-overview>`.
+For more information about modifying synced object schemas, see:
+:ref:`<synced-schema-overview>`.
 
 .. important:: Disable Development Mode for Production Apps
    
    Development mode is a development utility that is not suitable for 
    production use. Make sure that you turn off Development Mode before you 
    make your app accessible in a production environment.
 
+.. _dev-mode-breaking-changes:
+
+Breaking Changes
+````````````````
+
+App Services Apps in Development Mode that were created after September 13, 2022
+can make :ref:`breaking changes <breaking-change-quick-reference>` to synced
+object schemas from client code.
+
+**Prerequisites** 
+
+- App Services App created after September 13, 2022
```

Comment:
```suggestion
- App Services App created after September 13, 2023
```

I wish it was 2022 too Kyle. There's a couple other spots also with 2022

Reply:
Oh, dang. Thank you for catching that. You'd think I would've realized it's 2023 by... September. 😆